test/controllers/client/ClientContactDetailsSavedControllerSpec.scala

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package controllers.client
+
+import base.SpecBase
+import pages.ContactNamePage
+import play.api.test.FakeRequest
+import play.api.test.Helpers._
+import views.html.client.ClientContactDetailsSavedView
+
+class ClientContactDetailsSavedControllerSpec extends SpecBase {
+
+  "ClientContactDetailsSaved Controller" - {
+
+    "must return OK and the correct view for a GET when client contact details exist" in {
```

Comment:
change this test name to just `"must return OK and the correct view for a GET"` because there' is no dynamic content